certbot/certbot/_internal/plugins/manual.py

```diff
@@ -45,11 +45,14 @@ class Authenticator(common.Plugin):
         'list of all domains that are challenged for the current certificate.')
     _DNS_INSTRUCTIONS = """\
 Please deploy a DNS TXT record under the name
-{domain} with the following value:
+{domain}. with the following value:
```

Comment:
By making the root zone explicit, I hope it will lead to less confusion when people add the full hostname (without the trailing period) to their DNS zone editor and by that making the hostname relative to the origin of the zone editor.

I.e.: `_acme-challenge.example.com` would lead to actually being `_acme-challenge.example.com.example.com.` whereas `_acme-challenge.example.com.` would always be `_acme-challenge.example.com.`

Reply:
Oh, what a great catch! I've always entered subdomains relative to the root zone, so it wouldn't have occurred to me that people would try to put the whole thing in there. I read this change as a syntactic period, rather than a literal. Trying to think of how to make what we actually mean more clear, some thoughts, open to other suggestions:
- put the domain on a new line like we do with the validation
- quote it using `"`
- quote it using `

What do you think?

Reply:
Also, can we add this great explanation as a comment?

Reply:
Not sure if we should add even *more* text. I have the feeling that the more text is presented, the less chance there is it's actually read in total. Also it depends on the DNS zone editor if it's even *possible* to include the entire domain, as usually indeed one only adds the subdomain relative to the zones origin. An explanation with all kinds of ifs and buts *might* only confuse some people I think. Also, it's not *that* common on the Community, so I believe adding the period at the end could be enough. One might even reconsider adding the period in the first place, but I think it's more precise this way. 

Anyway, I agree it's a little bit weird now, so in the middle of the sentence, so I've shuffled the text a little bit to have it on a line of its own, so it's actually equal to the way the `http-01` text is presented.

Reply:
I meant as a code comment, for people who might modify this code in the future. I totally agree about minimizing text for the user!